config/config.json

```diff
@@ -25,23 +25,21 @@
         "config": {
           "*** You can add more hls.js settings here": "",
           "https://github.com/video-dev/hls.js/blob/master/docs/API.md": "",
-          "maxBufferLength": 30,
-          "maxMaxBufferLength": 600,
-          "maxBufferSize": 60000000,
+          "maxBufferLength": 6,
+          "maxMaxBufferLength": 6,
+          "maxBufferSize": 600000000,
           "maxBufferHole": 0.5,
           "lowBufferWatchdogPeriod": 0.5,
           "highBufferWatchdogPeriod": 3,
           "capLevelToPlayerSize": false,
           "dceHideManualQualityChoices": true,
           "debug": true,
           "testBandwidth" : false,
-          "initialQualityLevel": 4,
           "startLevel" : 4,
```

Comment:
HLS uses the startLevel not the initialQualityLevel as the initial quality level. removing old param